pkg/telemetry/prometheus/node.go

```diff
@@ -104,22 +102,11 @@ func Init(nodeID string, nodeType livekit.NodeType) error {
 		[]string{"type"},
 	)
 
-	promSysDroppedPacketPctGauge = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Namespace:   livekitNamespace,
-			Subsystem:   "node",
-			Name:        "dropped_packets",
-			ConstLabels: prometheus.Labels{"node_id": nodeID, "node_type": nodeType.String()},
-			Help:        "System level dropped outgoing packet percentage.",
-		},
-	)
```

Comment:
Was not used. Also, pct / sec messed with my head. More importantly, with multiple rate interval windows supported, unclear which one this should reflect. So, removed it.